source/slang/hlsl.meta.slang

```diff
@@ -12329,6 +12399,14 @@ uint4 WaveMatch(matrix<T,N,M> value)
     __target_switch
     {
     case hlsl: __intrinsic_asm "WaveMatch";
+    case glsl:
+    case cuda:
+    case spirv:
+        uint4 result = uint4(0xFFFFFFFF);
+        [ForceUnroll]
+        for (int i = 0; i < N; i++)
+            result &= WaveMatch(value[0]);
```

Comment:
Shouldn't this be like this?
```
result &= WaveMatch(value[i]);
```

Reply:
You are right. Thanks for catching this, will fix in a new PR.